compiler/passes/convert-uast.cpp

```diff
@@ -45,16 +46,24 @@ struct Converter {
   bool inTupleDecl = false;
 
   // Cache strings for special identifiers we might compare against.
```

Comment:
A future work item - we should figure out a better way to handle this kind of caching. Might be nice to have a TODO comment here.